doc/api/test.md

```diff
@@ -1318,13 +1318,18 @@ test('changes a mock behavior', (t) => {
 added:
   - v19.1.0
   - v18.13.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/49088
+    description: this now return the context as well as appending when not providing the onCall argument.
```

Comment:
```suggestion
    description: This now supports chaining calls.
```

Reply:
I wanted to emphasize that it append the implementation as it wasn't before, and instead it just replaced it